profilegen/src/profile_generator_options_parser.rb

```diff
@@ -0,0 +1,59 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
+require 'optparse'
+require 'ostruct'
+
+#
+# Command line options parser for the generation
+# of profile SDKs.
+#
+# Options
+#    -h, --help : Displays help for the profile generator
+#    -o, --output_dir : Output directory to store the generated profile SDK
```

Comment:
please add mark each required v/s option arguments. 

We should have default output_dir...the less options the better